db/seeds/utils.js

```diff
@@ -20,3 +22,17 @@ exports.formatComments = (comments, idLookup) => {
     };
   });
 };
+
+exports.articleAddComments = (article) => {
+  return db.query(`
+    SELECT * FROM comments
+    WHERE comments.article_id = ${article.article_id}
+  `).then((comments) => {
+    article.comments = comments.rows;
+    article.comment_count = comments.rows.length;
+    delete article.body;
```

Comment:
You shouldn't need to delete the body - think about how you could change your query to not have the body return?